util/Setup/EnvironmentFileBuilder.cs

```diff
@@ -51,6 +51,12 @@ public void BuildForUpdater()
             _globalOverrideValues.Remove("globalSettings__pushRelayBaseUri");
         }
 
+        if (_globalOverrideValues.TryGetValue("globalSettings__baseServiceUri__vault", out var vaulturi) && vaulturi != _context.Config.Url)
```

Comment:
Probably should have mentioned this yesterday.

```suggestion
        if (_globalOverrideValues.TryGetValue("globalSettings__baseServiceUri__vault", out var vaultUri) && vaultUri != _context.Config.Url)
```